.github/workflows/license-scan.yml

```diff
@@ -0,0 +1,34 @@
+name: Blackduck - PR Analysis
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  black-duck:
+    runs-on: ubuntu-latest
+    env:
+      # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets
+      BLACKDUCK_URL: ${{ secrets.BLACKDUCK_URL }}
+    steps:
+      - name: Checkout
+        if: ${{ env.BLACKDUCK_URL != '' }}
+        uses: actions/checkout@v2
+      - name: Set up Java 11
+        if: ${{ env.BLACKDUCK_URL != '' }}
+        uses: actions/setup-java@v2
+        with:
+          java-version: "11"
+          distribution: "adopt"
+      - name: Run Synopsys Detect
+        if: ${{ env.BLACKDUCK_URL != '' }}
+        uses: synopsys-sig/detect-action@v0.3.0
```

Comment:
this is probably the only critical thing with potential security implications:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
As the creator is verified: 
https://github.com/marketplace?type=&verification=&query=synopsys-sig+
it should be fine. 
@FireyFly maybe you can you link the example you used on our blackduck wiki page?

Reply:
Did so, added pointers to each of our configs which one was used as the basis to make it more clear.